vulnerabilities/api.py

```diff
@@ -175,9 +176,9 @@ def filter_purl(self, queryset, name, value):
 
 
 class PackageViewSet(viewsets.ReadOnlyModelViewSet):
-    queryset = Package.objects.all()
+    queryset = Package.objects.all().order_by("id")
```

Comment:
Why this change? If there is an issue with the ordering, please create a new issue and discuss the changes first.
This may need to be implement on the Model instead of on the API side. But the is no explanation about why you are changing all the ordering in the API context.

Reply:
@tdruez there is an issue reported here for same https://github.com/nexB/vulnerablecode/issues/685, but we can tackle this in a separate PR, will remove unrelated changes

Reply:
1 issue -> 1 PR.

Also, this fix approach for #685 seems wrong.

Reply:
Alright! will fix this by changing in models side not at the API side